src/main/java/racingcar/domain/Tracker.java

```diff
@@ -0,0 +1,47 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Tracker {
+
+    private final List<Record> records = new ArrayList<>();
+
+    public void saveRecords(Cars cars) {
+        if (cars.isEmpty()) {
+            throw new IllegalStateException("자동차는 한 대 이상이어야 기록됩니다.");
+        }
+
+        cars.saveRecords(records);
+    }
+
+    public List<Record> findWinnerRecords() {
+
+        if (records.isEmpty()) {
+            throw new IllegalStateException("기록이 있어야 우승자를 찾을 수 있습니다.");
+        }
+
+        int maxPosition = findMaxPosition();
+
+        return records.stream()
+                .filter(record -> record.getPosition() == maxPosition)
+                .collect(Collectors.toList());
+    }
+
+    private int findMaxPosition() {
+        int maxPosition = 0;
+        for (Record record : records) {
+            final int position = record.getPosition();
+            if (position > maxPosition) {
+                maxPosition = position;
+            }
+        }
+
+        return maxPosition;
```

Comment:
다른 곳에는 stream API 잘 적용해 주셨는데, 사실 이곳이야 말로 stream API 적용하기 최적인 곳입니다. 👍 
한번 적용 부탁드립니다.

Reply:
좋은 의견 감사합니다! 아래 처럼 적용했습니다

```suggestion
    private int findMaxPosition() {
        return records.stream()
                .mapToInt(Record::getPosition)
                .max()
                .orElse(0);
    }
```